SU2_CFD/src/drivers/CSinglezoneDriver.cpp

```diff
@@ -209,11 +209,13 @@ void CSinglezoneDriver::Output(unsigned long TimeIter) {
 #else
   StartTime = MPI_Wtime();
 #endif
-    
+
+  bool force_write = StopCalc || !config_container[ZONE_0]->GetTime_Domain();
+
```

Comment:
@talbring , is it ok if I do this? I was not getting any file output for discrete adjoint fea cases (I think it would happen for any unconverged case in general) Let me know if there is a better way.

Reply:
Hm but this routine should not be called if the case is not converged anyway. The output routines in in CIteration are responsible for intermediate writing.

Reply:
It is not being handled by single zone adjoint in the Run method and the SecondaryRecording method, the latter being required for output of sensitivities.